app/views/discussions/recent.html.haml

```diff
@@ -15,10 +15,12 @@
            object: Discussion.recently_created,
            locals: { nothing: t('.no_discussions') }
 
-  -# TODO FIXME show 5 discussions with the latest comments in them
-  -#%h4.text-center
-  -#  = t '.recently_commented_title'
-  -#
-  -#= render partial: 'list',
-  -#         object: Discussion.recently_commented,
-  -#         locals: { nothing: t('.no_discussions') }
+  - recently_commented = Discussion.recently_commented.all
```

Comment:
Can be. Then, however, we need to put it in the locales or set two @ instance variables. My IDE warns me that having two such variables is a bad practice. Locales, on the other hand, I kind of dislike :).